Userland/Utilities/hexdump.cpp

```diff
@@ -67,43 +63,42 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         putchar('\n');
     };
 
-    Array<u8, BUFSIZ> contents;
-    Span<u8> previous_line;
-    static_assert(LINE_LENGTH_BYTES * 2 <= contents.size(), "Buffer is too small?!");
```

Comment:
I added it back as a static_assert. If I use `VERIFY`, clang-tidy complains that it should use static_assert instead.

I'm a bit confused about what this is checking. We're defining `LINE_LENGTH_BYTES` as 16, then declaring a 1024 byte array, and then asserting that 16 * 2 < 1024 ?  It seems a bit silly to assert this, but maybe I'm missing something. Is the check just in case someone decides to redefine `LINE_LENGTH_BYTES` to be 513 or more bytes?

Reply:
My bad, you're right with the `static_assert`.

To be honest, I didn't look too long to what it was asserting. Just that your changes wasn't conflicting with the check.
I mean, this might be useless, but I find it a bit silly to remove it silently.

Anyway, we will see when a maintainer will review it.

Reply:
That `static_assert` is indeed a bit bizarre but I guess it's not hurting anybody either :)